README.md

```diff
@@ -1,4 +1,59 @@
-# Python Boilerplate For GitHub Tasks
+# Burger Recipe
 
 **Please note:** read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md)
 before starting.
+
+You work at the In-N-Out and make delicious burgers. 
+
+Create the `BurgerRecipe` class, its constructor should take and save the number of ingredients needed to make a burger, such as cheese, tomatoes, cutlets, eggs, buns, and sauce.
+
+```python
+cheese_burger = Burger(buns=2, chesse=2, tomatoes=1, cutlets=1, eggs=1, sauce="ketchup")
+```
+
+You can make burgers with different number of ingredients and different sauces, for example:
+
+- `buns` - can range from 2 to 3;
+- `cheese` - can range from 0 to 2;
+- `tomatoes` - can range from 0 to 3;
+- `cutlets` - can range from 1 to 3;
+- `eggs` - can range from 0 to 2;
+- `sauce` - can be ketchup, mayo, or burger.
+
+So, it would be convenient to use descriptors to make sure that number of ingredients is `int` in a range mentioned above
+and sauce is one of ketchup, mayo, or burger.
+
+The main task is to create the `BurgerRecipe` class, and for convenience you should consider such descriptors.
+
+The `Validator` class that should have such methods:
+
+- the `__set_name__` method, which takes the name of the attribute, adds `_` to its beginning and stores it in the `protected_name` attribute;
+- the `__get__` method that returns the attribute value;
+- the `__set__` method, which sets the attribute value (**note:** there should be no validation implemented);
+- the `validate` abstract method which takes value.
+
+Also, you need to implement the `Number` class, which is the `Validator` child class. It should have:
+
+- the `__init__` method which takes and stores `min_value` and `max_value`;
+- the `validate` method which should check the value type, and if the type is incorrect it must raise the `TypeError` exception with the message `Quantity should be integer`. 
```

Comment:
- the `validate` method, which checks the value type, and if the type is incorrect, it must raise the `TypeError` exception with the `Quantity should be integer` message.